benchmark/scripts/Benchmark_Driver

```diff
@@ -346,17 +350,40 @@ def positive_int(value):
 
 
 def main():
-    parser = argparse.ArgumentParser(description='Swift benchmarks driver')
-    subparsers = parser.add_subparsers()
-
-    submit_parser = subparsers.add_parser(
-        'submit',
-        help='run benchmarks and submit results to LNT')
-    submit_parser.add_argument(
+    parser = argparse.ArgumentParser(
+        epilog='Example: ./Benchmark_Driver run -i 5 -f Array'
+    )
+    subparsers = parser.add_subparsers(
+        title='Swift benchmark driver commands',
+        help='See COMMAND -h for additional arguments', metavar='<command>')
+
+    parent_parser = argparse.ArgumentParser(add_help=False)
+    benchmarks_group = parent_parser.add_mutually_exclusive_group()
+    benchmarks_group.add_argument(
+        'benchmark',
+        default=[],
+        help='benchmark to run (default: all)', nargs='*', metavar="BENCHMARK")
+    benchmarks_group.add_argument(
+        '-f', '--filter',
+        help='run all tests whose name starts with PREFIX', metavar="PREFIX")
+    parent_parser.add_argument(
         '-t', '--tests',
         help='directory containing Benchmark_O{,none,unchecked} ' +
         '(default: DRIVER_DIR)',
         default=DRIVER_DIR)
+
+    submit_parser = subparsers.add_parser(
+        'submit',
+        help='Run benchmarks and submit results to LNT',
+        parents=[parent_parser])
+    submit_parser.add_argument(
+            '-o', '--optimization', nargs='+',
+            help='optimization levels to use (default: O Onone Ounchecked)',
+            default=['O', 'Onone', 'Ounchecked'])
+    submit_parser.add_argument(
+        '-i', '--iterations',
```

Comment:
I am going to ignore this for now, but we should really change this argument to --samples. In terms of the lower level benchmark driver, iterations is proportional to the length of a sample. This is really changing the number of individual samples that we are running.